iceberg/iceberg-handler/src/test/queries/positive/iceberg_rest_catalog.q

```diff
@@ -0,0 +1,37 @@
+-- SORT_QUERY_RESULTS
+-- Mask neededVirtualColumns due to non-strict order
+--! qt:replace:/(\s+neededVirtualColumns:\s)(.*)/$1#Masked#/
+-- Mask random uuid
+--! qt:replace:/(\s+'uuid'=')\S+('\s*)/$1#Masked#$2/
+-- Mask a random snapshot id
+--! qt:replace:/(\s+current-snapshot-id\s+)\S+(\s*)/$1#Masked#/
+-- Mask added file size
+--! qt:replace:/(\S\"added-files-size\\\":\\\")(\d+)(\\\")/$1#Masked#$3/
+-- Mask total file size
+--! qt:replace:/(\S\"total-files-size\\\":\\\")(\d+)(\\\")/$1#Masked#$3/
+-- Mask current-snapshot-timestamp-ms
+--! qt:replace:/(\s+current-snapshot-timestamp-ms\s+)\S+(\s*)/$1#Masked#$2/
+--! qt:replace:/(MAJOR\s+succeeded\s+)[a-zA-Z0-9\-\.\s+]+(\s+manual)/$1#Masked#$2/
+--! qt:replace:/(MAJOR\s+refused\s+)[a-zA-Z0-9\-\.\s+]+(\s+manual)/$1#Masked#$2/
+-- Mask compaction id as they will be allocated in parallel threads
+--! qt:replace:/^[0-9]/#Masked#/
+-- Mask removed file size
+--! qt:replace:/(\S\"removed-files-size\\\":\\\")(\d+)(\\\")/$1#Masked#$3/
+-- Mask iceberg version
+--! qt:replace:/(\S\"iceberg-version\\\":\\\")(\w+\s\w+\s\d+\.\d+\.\d+\s\(\w+\s\w+\))(\\\")/$1#Masked#$3/
+
+create table ice_orc (
+    first_name string, 
+    last_name string,
+    dept_id bigint,
+    team_id bigint
+ )
+partitioned by (company_id bigint)
+stored by iceberg stored as orc;
+
+--! Verify rest catalog properties are set in conf
+set iceberg.catalog;
```

Comment:
1. where is that set? 
2. shouldn't we use CATALOG_DEFAULT('metastore.catalog.default')?
3. could we please add a test when catalog is explicitly set on a table props: 
````
CREATE EXTERNAL TABLE ice_orc2 (
    orderid INT, 
    quantity INT, 
    itemid INT, 
    tradets TIMESTAMP
)
PARTITIONED BY (p1 STRING, p2 STRING)
STORED BY ICEBERG STORED AS ORC
TBLPROPERTIES('format-version'='2', 'iceberg.catalog'='ice01');
````